consul/content.md

```diff
@@ -5,6 +5,9 @@ Consul is a distributed, highly-available, and multi-datacenter aware tool for s
 -	[Consul documentation](https://www.consul.io/)
 -	[Consul on GitHub](https://github.com/hashicorp/consul)
 
+## Notice for (x.y.z-ent) Image Variants
+This HashiCorp enterprise software is for use only by customers who have a valid and active license agreement with HashiCorp or an authorized HashiCorp reseller. If you do not have a valid license to use this software, you may not download or otherwise use it. All use of this software is subject to the terms and conditions of your license agreement, and all other rights are expressly reserved.
```

Comment:
Continuing my comment from https://github.com/docker-library/official-images/pull/5378 (now with the added clarification that `consul:1.4.2-ent` is intended to stay after https://github.com/docker-library/official-images/pull/5378 backfills older versions which then get dropped), it still seems strange to me to include these here if their use is intended to be controlled, limited, or even discouraged, and it seems like a separate `consul/ent` repo would make more sense (and be more easily controlled), or even taking advantage of Docker Store's ability to control image distribution of images/repos via licensing / permissions?